pom.xml

```diff
@@ -83,7 +83,7 @@
 
         <scalatest.version>3.0.3</scalatest.version>
         <iceberg.name>iceberg-spark3-runtime</iceberg.name>
-        <iceberg.version>0.11.0</iceberg.version>
+        <iceberg.version>0.11.1</iceberg.version>
```

Comment:
Would you mind if you can test this profile locally to see if the incompatibility of Iceberg has been gone ?https://github.com/yaooqinn/kyuubi/blob/master/.github/workflows/master.yml#L37 + remove -Dmaven.plugin.scalatest.exclude.tags=org.apache.kyuubi.tags.DataLakeTest

Reply:
Iceberg not support Spark 3.1.1 yet, see https://github.com/apache/iceberg/issues/2311

Reply:
make sense